nvm.sh

```diff
@@ -541,7 +540,7 @@ nvm_remote_versions() {
   if [ -z "${NVM_LTS-}" ] && {
     [ -z "${NVM_FLAVOR-}" ] || [ "${NVM_FLAVOR-}" = "${NVM_IOJS_PREFIX}" ];
   }; then
-    NVM_LS_REMOTE_IOJS_OUTPUT=$(nvm_ls_remote_iojs "${PATTERN-}") &&:
+    NVM_LS_REMOTE_IOJS_OUTPUT=$(nvm_ls_remote_iojs "${PATTERN-}") && :
```

Comment:
`&&:` is an intentional, googleable thing - `&& :` is less clear as to the original intentions.